gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,6 @@
-#Thu Jun 30 16:37:08 PDT 2016
+#Sat Oct 03 17:00:57 PDT 2015
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-2.13-bin.zip
```

Comment:
IIRC, this is an auto-gen file for including `gradlew` in a repo. I was just rolling back to what's on master to avoid the CI errors that were previously plaguing this PR.

The reason for upgrading `gradlew` were driven by the choice of `protobuf-gradle-plugin` 0.7.7, which is now 0.7.3 as you did in your PR, so it doesn't seem like `gradlew` needs to be updated in this PR either.